api/app/Exceptions/BadRequestException.php

```diff
@@ -0,0 +1,17 @@
+<?php namespace App\Exceptions;
+
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class BadRequestException extends HttpException
+{
+    /**
+     * Create a new Bad Request exception instance.
+     *
+     * @param  string  $message
+     * @return void
+     */
+    public function __construct($message = 'Bad Request.')
```

Comment:
Extended exceptions should match the parameters of the base `\Exception` class at minimum, ie add `$message = "", $code = 0, Exception $previous = null` to the params